main/src/cgeo/geocaching/CacheListActivity.java

```diff
@@ -743,14 +743,20 @@ public boolean onOptionsItemSelected(final MenuItem item) {
                 invalidateOptionsMenuCompatible();
                 return true;
             case R.id.menu_refresh_stored:
-                Dialogs.confirmYesNo(this, R.string.caches_refresh_all, R.string.caches_refresh_all_warning, new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(final DialogInterface dialog, final int id) {
-                        refreshStored(adapter.getCheckedOrAllCaches());
-                        invalidateOptionsMenuCompatible();
-                        dialog.cancel();
-                    }
-                });
+                final boolean isOffline = type == CacheListType.OFFLINE;
```

Comment:
Is it really worth it, to introduce an intermediate variable just for this check?